### ISSUE TITLE
MessageBox.Show deadlock after WebView2 init

### ISSUE THREAD

**Issue by AbeniMatteo** [MicrosoftEdge/WebView2Feedback#877]
**Description**
MessageBox window does not appear and the process increases CPU usage.
Possibly related to #329.

**Version**
SDK: 1.0.781-prerelease
Runtime: 89.0.774.8 (Official build) dev (64-bit)
Framework: .NET 5.0.102 + WPF
OS: Win10 10.0.19042 Build 19042

**Repro Steps**
Run the following code to see the first message displayed correctly and the second call to `MessageBox.Show` (after `new WebView2`) deadlocking:

Program.cs
```cs
namespace Repro
{
    using System;
    using System.Windows;
    using Microsoft.Web.WebView2.Wpf;

    static class Program
    {
        [STAThread]
        static void Main()
        {
            // Init Application.Current
            new Application();
            Application.Current.ShutdownMode = ShutdownMode.OnExplicitShutdown;

            Test(initWebView: false);
            Test(initWebView: true);
        }

        static void Test(bool initWebView)
        {
            var w = new Window();
            w.Loaded += (sender, e) =>
            {
                if (initWebView)
                    w.Content = new WebView2();

                Application.Current.Dispatcher.BeginInvoke((Action)(() =>
                {
                    MessageBox.Show("Test");
                    w.Close();
                }));
            };
            w.ShowDialog();
        }
    }
}
```

Repro.csproj
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net5.0-windows</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.781-prerelease" />
  </ItemGroup>
</Project>
```

**Comment by AbeniMatteo** [comment]
High CPU usage is not specific to this problem, check #878.

**Comment by AbeniMatteo** [comment]
@champnic The `Begin\EndPaint` [workaround](https://github.com/MicrosoftEdge/WebView2Feedback/issues/878#issuecomment-772883632) also fixes this problem. I will wait the next version to close this issue.

**Comment by champnic** [comment]
Thanks for verifying @AbeniMatteo! I'll let you know when we release the fix.

**Comment by AbeniMatteo** [comment]
Fixed with `1.0.790-prerelease`.
